tests/api/test_get_status.py

```diff
@@ -10,26 +15,48 @@ def test_get_status(client):
                                                                     'version': 'latest'}}}
 
 
-def test_ready(client):
+def test_ready(client, minio):
     response = client.post('/start-job', content_type='application/json', data=json.dumps({
-        'job_id': 'uuid-5',
+        'job_id': 'uuid-ready',
         'sheep_id': 'bare_sheep',
         'model': {
             'name': 'model_1',
             'version': 'latest'
         },
         'payload': 'Payload content'
     }))
-
     assert response.status_code == 200
+    minio.put_object('uuid-ready', DONE_FILE, BytesIO(), 0)
+    timestamp = datetime.now()
 
-    response = client.get('/jobs/uuid-5/wait_ready')
+    response = client.get('/jobs/uuid-ready/wait_ready')
     assert response.status_code == 200
     assert response.json == {'ready': True}
 
-    response = client.get('/jobs/uuid-5/ready')
+    response = client.get('/jobs/uuid-ready/ready')
     assert response.status_code == 200
-    assert response.json == {'ready': True}
+    assert response.json['ready'] is True
+    assert response.json['finished_at'][:17] == timestamp.isoformat()[:17]
```

Comment:
Why the slice?

Reply:
Cause the timestamps can be only approximately same, so I sliced off seconds and fractions of seconds

Reply:
That's... not pretty. Please, at least calculate the difference and assert it is less than a second or something.

Reply:
@gdynusa Btw what would happen if we run the tests around midnight (which circleci does everyday...)